xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/http/JsonMapper.java

```diff
@@ -53,9 +51,7 @@ public String writeValueAsString(Object value) {
         return objectMapper.convertValue(value, klass);
     }
 
-    private SimpleModule buildPsd2DateTimeDeserializerModule() {
-        SimpleModule dateTimeModule = new SimpleModule();
-        dateTimeModule.addDeserializer(OffsetDateTime.class, new Psd2DateTimeDeserializer());
-        return dateTimeModule;
+    public ObjectMapper getNewInstance() {
```

Comment:
The whole point of this class was to avoid explicit dependency on jackson's ObjectMapper. Now it's exposed via a getter that's named as a mix of two static factory method names.

Reply:
the new interface was introduced